src/qpython_imageprovider.cpp

```diff
@@ -141,12 +145,12 @@ QPythonImageProvider::requestImage(const QString &id, QSize *size, const QSize &
     // While we could re-pack the data to be aligned, we don't want to do that
     // for performance reasons.
     // If we're using 32-bit data (e.g. ARGB32), it will always be aligned.
-    if (format != -1 && bitsPerPixel != 32) {
+    if (format > 0 && bitsPerPixel != 32) {
```

Comment:
This changes behavior when `format == 0`, maybe use `>= 0`, or maybe it's unused?